lib/node_modules/@stdlib/utils/every-by/docs/types/index.d.ts

```diff
@@ -20,7 +20,7 @@
 
 /// <reference types="@stdlib/types"/>
 
-import { Collection } from '@stdlib/types/array';
+import { Collection } from '@stdlib/types/object';
```

Comment:
Need to revert this change:

```suggestion
import { Collection } from '@stdlib/types/array';
```